mcux/drivers/imx/CMakeLists.txt

```diff
@@ -34,7 +34,10 @@ zephyr_library_sources_ifdef(CONFIG_SPI_MCUX_LPSPI	fsl_lpspi.c)
 zephyr_library_sources_ifdef(CONFIG_UART_MCUX_LPUART	fsl_lpuart.c)
 zephyr_library_sources_ifdef(CONFIG_VIDEO_MCUX_CSI	fsl_csi.c)
 zephyr_library_sources_ifdef(CONFIG_WDT_MCUX_IMX_WDOG    fsl_wdog.c)
-zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_FLEXCAN	fsl_flexcan.c)
+zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_ADC12  fsl_adc.c)
+zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_ADC16	fsl_adc.c)
+zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_LTADC	fsl_adc.c)
+zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_RTADC	fsl_adc.c)
```

Comment:
The only change in this file should be adding one line:
```
zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_RTADC	fsl_adc.c)
```

Reply:
Update with force push